hazelcast/src/main/java/com/hazelcast/multimap/impl/MultiMapDataSerializerHook.java

```diff
@@ -101,6 +102,7 @@
     public static final int TXN_REMOVE_ALL_BACKUP = 38;
     public static final int TXN_ROLLBACK = 39;
     public static final int TXN_ROLLBACK_BACKUP = 40;
+    public static final int MULTIMAP_OP_FACTORY = 41;
 
 
```

Comment:
Next time when you do such a conversion PR, please split them up in logical units. E.g. map conversion PR and multimap conversion PR. I have burned my fingers so many times on a PR that is bigger than it needs to be and the effect is that quite a lot of time (and patience) is being wasted.